Sources/Sentry/include/SentryTracer.h

```diff
@@ -43,6 +43,13 @@ static const NSTimeInterval SENTRY_AUTO_TRANSACTION_MAX_DURATION = 500.0;
 
 @property (nullable, nonatomic, copy) BOOL (^shouldIgnoreWaitForChildrenCallback)(id<SentrySpan>);
 
+/**
+ * This flag indicates whether the trace should be captured when the timeout triggers.
+ * If Yes, this tracer will be discarced in case the timeout triggers.
+ * Default @c NO
+ */
+@property (nonatomic) BOOL finishMustBeCalled;
```

Comment:
`m`: Any good reason why this can't be part of `SentryTracerConfiguration`? Logically it would make sense to me to add it there.

Reply:
Ah, I guess because we need to set the hub to nil and call all the callback, right?

Reply:
> Any good reason why this can't be part of SentryTracerConfiguration? Logically it would make sense to me to add it there

Actually because, `SentryPerformanceTracker` is the one creating the tracer, but `SentryUIViewControllerPerformanceTracker` is the class that knows those tracers need that flag.
I guess we could add an extra argument in `SentryPerformanceTracker`.